spec/models/armor_spec.rb

```diff
@@ -312,6 +318,11 @@
         it "doesn't remove the existing enchantments" do
           expect(armor.enchantments.reload.length).to eq 4
         end
+
+        it 'sets "added_automatically" only on the new associations' do
+          expect(armor.enchantables_enchantments.pluck(:added_automatically))
+            .to eq [true, true, false, false]
```

Comment:
Because of how the tests are set up, the automatically added ones will be first.